src/odfesqlodbc/es_parse_result.cpp

```diff
@@ -329,19 +337,18 @@ bool AssignRowData(const json_arr_it &row, const schema_type &row_schema,
 
     // Loop through and assign data
     size_t i = 0;
-    for (auto row_column = row.value_begin(); i < row_schema.size();
+    for (auto row_column = row.value_begin(); i < row_schema_size;
          ++row_column, ++i) {
         if (row_column->is_null()) {
             tuple[i].len = SQL_NULL_DATA;
             tuple[i].value = NULL;
         } else {
             // Copy string over to tuple
-            const std::string data = row_column->str();
-            tuple[i].len = static_cast< int >(data.length());
+            tuple[i].len = static_cast< int >(row_column->str().length());
```

Comment:
What does str() return? Does it return a const reference or a copy? I only ask b/c if it returns a copy we've now moved to 3 copies where before there was one. If it's a const reference then we're good.

Reply:
Thanks for noticing. It creates a stream/buffer and returns a copy. I will revert this change to keep only one copy.

Reply:
done